Dockerfile

```diff
@@ -14,9 +14,12 @@ RUN npm install
  
 # Copies everything over to Docker environment
 COPY . .
- 
-# Uses port which is used by the actual application
-EXPOSE 4100
- 
-# Finally runs the application
-CMD [ "npm", "start" ]
+
+# Build the React application
+CMD [ "npm", "run", "build" ]
```

Comment:
to get this dockerfile working w/ podman it looks like we need to replace this line with:

`RUN npm run build`